api/web_components/index.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: default
+title: Web Components
+parent: API
+nav_order: 20
+has_children: true
+---
+# Data Commons Web Components
+
+Embed [Data Commons](https://datacommons.org)
+[statistical variable](https://datacommons.org/tools/statvar) observation
+visualizations in your web application.
+
+<img src="/assets/images/web_components/all.png" style="width: 100%; max-width: 1024px;">
+
+## Usage
+
+Include `datacommons.js` in your html's
+`<head>...</head>` tag. Then use `datacommons-*` [web
```

Comment:
I think datacommons-* is a little confusing here.. how about:
```suggestion
`<head>...</head>` tag. Then use Data Commons [web component](...)  tags (e.g. `datacommons-line`)
```

Reply:
Sg! Used your suggestion.